spec/unit/railtie_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+require 'mongo_mapper/railtie'
+
+describe MongoMapper::Railtie do
+  def with_rails(mongo_yml, &block)
+    old_env, Rails.env = Rails.env, "test"
+    old_time_zone_default = Time.zone_default
+
+    Dir.mktmpdir do |rails_root|
+      Dir.mkdir("#{rails_root}/config")
+      File.write("#{rails_root}/config/mongo.yml", mongo_yml)
+
+      fake_app = Module.new {
+        def self.name
+          'FakeApp'
+        end
+
+        const_set(:Application, Class.new(::Rails::Application) {
+          config.load_defaults "#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}" if config.respond_to?(:load_defaults)
+          config.eager_load = false
+          config.root = rails_root
+        })
+      }
+      fake_app::Application.initialize!
+
+      block.call
+    end
+  ensure
+    Rails.application = nil
+    Rails.app_class = nil
+    Rails.env = old_env
+    Time.zone_default = old_time_zone_default
```

Comment:
We need this line to pass https://github.com/mongomapper/mongomapper/blob/716a17783badac9de3e1820f4869c919f70937d2/spec/unit/time_zones_spec.rb#L17-L22.

Reply:
Um, Rails app initialization changes other ActiveSupport configurations. It can cause unintentional behaviors.